src/status_im/contexts/settings/wallet/keypairs_and_accounts/view.cljs

```diff
@@ -0,0 +1,105 @@
+(ns status-im.contexts.settings.wallet.keypairs-and-accounts.view
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
+            [quo.theme]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im.constants :as constants]
+            [status-im.contexts.settings.wallet.keypairs-and-accounts.actions.view :as actions]
+            [status-im.contexts.settings.wallet.keypairs-and-accounts.style :as style]
+            [utils.address :as utils]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn navigate-back
+  []
+  (rf/dispatch [:navigate-back]))
+
+(defn- parse-accounts
+  [given-accounts]
+  (->> given-accounts
+       (keep (fn [{:keys [path customization-color emoji name address]}]
+               (when (not (string/starts-with? path constants/path-eip1581))
+                 {:account-props {:customization-color customization-color
+                                  :size                32
+                                  :emoji               emoji
+                                  :type                :default
+                                  :name                name
+                                  :address             address}
+                  :networks      []
+                  :state         :default
+                  :action        :none})))))
+
+(defn on-options-press
+  [{:keys [theme]
+    :as   props}]
+  (rf/dispatch [:show-bottom-sheet
+                {:content (fn [] [actions/view props])
+                 :theme   theme}]))
+
+(defn- keypair
+  [{:keys [accounts name]}
+   index _
+   {:keys [profile-picture compressed-key customization-color]}]
+  (let [theme            (quo.theme/use-theme)
+        accounts         (parse-accounts accounts)
+        default-keypair? (zero? index)
+        shortened-key    (when default-keypair?
+                           (utils/get-shortened-compressed-key compressed-key))
+        on-press         (rn/use-callback
+                          (fn []
+                            (on-options-press
+                             (merge {:theme theme
```

Comment:
Hmm I'm not sure either, do you mind showing how you would use `cond->` in this case? I imagine that we could use `cond->` with `assoc` to conditionally merge stuff, but I'm not sure if that's what you're mentioning.

Reply:
Sorry for not being clearer. Exactly that way @seanstrom, `cond->` followed by `assoc`.

Reply:
It's not as elegant for sure, and I love `merge`. It's just that in recent times we've been trying to avoid `merge` unless the keys to merge into the default value are unknown/dynamic. So I extended this idea to code like the one you wrote. But it's no big deal, more like curiosity in case you knew.